R/adapter.r

```diff
@@ -245,9 +245,15 @@ construct_s3_adapter <- function() {
       }
     }
 
-    # If the user provided an s3 path, like "s3://somebucket/some/path/", 
+    # If the user provided an s3 path, like "s3://somebucket/some/path/",
+    # or wants to overwrite an already existing file by providing an option,
     # pass it along to the s3read function.
-    args <- list(obj = object, name = opts$resource)
+    args <- list(obj = object, name = opts$resource, safe = FALSE)
+    # Set `safe = FALSE` by default for backwards compatibility.
+    # Since running syberia files is a reproducible process there
+    # shouldn't be a lot of damage done
+    # There is an option to overwrite this behavior
+    if (is.element('safe', names(opts))) args$safe <- opts$safe
```

Comment:
Just to have the same style in two adjacent lines
```
if (is.element('safe', names(opts))) args$safe <- opts$safe
if (is.element('s3path', names(opts))) args$.path <- opts$s3path
```
Should I refactor both before merge?